src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,9 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
That's basically because it felt weird to not be able to do that. So if you open the search in regular view, you can see your favourites and the one that's selected so being able to still select those favourites seems to make sense. It's easily removed if this is just my opinion though.